pyproject.toml

```diff
@@ -33,6 +33,7 @@ py-version="3.8"
 disable = [
     "missing-module-docstring",
     "too-few-public-methods",
+    "no-self-use",
```

Comment:
Shouldn't we decorate with `staticmethod` instead?

Reply:
I think this is an annoying pylint's message that we're going to make optional soon, but yeah that's the fix 😄

Reply:
But this isn't a false positive right?

Reply:
I think it happened when ``def treat_token(self, tokeninfo: TokenInfo) -> TokenInfo:`` in the OptionalFormatter wasn't the same than in the base class (during dev), it disappeared now.